src/ape/api/networks.py

```diff
@@ -606,6 +607,18 @@ def config(self) -> PluginConfig:
     def _network_config(self) -> Dict:
         return self.config.get(self.name, {})
 
+    @cached_property
+    def gas_limit(self) -> Union[Literal["auto"], Literal["max"], int]:
+        value = self._network_config.get("gas_limit", "auto")
+
+        if value.isdigit():
+            return int(value)
+
+        if value not in ("auto", "max"):
```

Comment:
thoughts on:

```suggestion
        if value.lower() not in ("auto", "max"):
```

so that if a particular loud config file uses `MAX`, it will still work

Reply:
Yeah definitely agree, allowing case insensitivity is always good to me since the meaning of the values is the same

Reply:
Removed `lower()` here in 4c502af781eca05c73bfa7299393040f58d2038f just because the `NetworkConfig` validator now accounts for it